### PR TITLE
feat(medusa): add pricing integration feature flag

### PR DESCRIPTION
**Pull request by pKorsholm**

### CODE REVIEW

File: packages/medusa/src/loaders/feature-flags/pricing-integration.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { FlagSettings } from "../../types/feature-flags"
+
+const PricingIntegrationFeatureFlag: FlagSettings = {
+  key: "pricing_integration",
+  default_val: false,
+  env_key: "MEDUSA_FF_PRICING_INTEGRATION",
```
Comment (riqwan) [inline packages/medusa/src/loaders/feature-flags/pricing-integration.ts]: Maybe pricing module integration? Or with_pricing_module? Or just pricing_module

**Review by riqwan — APPROVED** [review]
Made a suggestion, I think module should be explicit in there. Lgtm

File: packages/medusa/src/loaders/feature-flags/pricing-integration.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { FlagSettings } from "../../types/feature-flags"
+
+const PricingIntegrationFeatureFlag: FlagSettings = {
+  key: "pricing_integration",
+  default_val: false,
+  env_key: "MEDUSA_FF_PRICE_MODULE",
```
Comment (olivermrbl) [inline packages/medusa/src/loaders/feature-flags/pricing-integration.ts]: suggestion(non-blocking): The key and env_key are usually very similar. We could also use one that would be consistent with the Product Module isolation flag?
```suggestion
  key: "isolate_pricing_domain",
  default_val: false,
  env_key: "MEDUSA_FF_ISOLATE_PRICING_DOMAIN",
```

**Review by olivermrbl — APPROVED** [review]

**Comment by olivermrbl** [comment]
Feel free to merge as you please :)

**Pull request merged**